monero-outreach-docs/translations/pt-br/storyboard/tool_for_renewal_pt_br.md

```diff
@@ -26,51 +26,51 @@ Nós não esperamos que você aceite nossas palavras por nada disso e encorajamo
 
 ### _Leitura Complementar:_
```

Comment:
Hey @nelsonrenan 

I'm not sure what happened here but the headers are not showing up. Can you please check the file?

Reply:
Never mind, I had an issue with my browser.